test/user.test.js

```diff
@@ -67,6 +68,26 @@ describe('User', function() {
       });
     });
 
+   it('Create a new user (email case-sensitivity off)', function(done) {
+      User.settings.caseSensitiveEmail = false;
+      User.create({email: 'F@b.com', password: 'bar'}, function(err, user) {
+        assert(!err);
```

Comment:
I followed the same conventions at the other tests in that `describe` block. Should I replace `assert(!err)` in all cases then?